db/seeds.rb

```diff
@@ -5,18 +5,27 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
-Item.create(name: 'Lasagna', description: "Just like your granny made it", price: 7.99, category_id: 1, image_url: "http://cdn.thedailybeast.com/content/dailybeast/cheats/2013/02/09/french-gov-vows-to-punish-horsemeat-pushers/jcr:content/image.crop.800.500.jpg/45347925.cached.jpg")
+Item.create(name: 'Pizza', description: "Every pie wishes it was deep dish", price: 11.99, category_id: 1, image_url: "http://freehdw.com/images/800/pizza-wide.jpg")
 Item.create(name: 'Burger', description: "Beef city, population: you", price: 6.99, category_id: 1, image_url: "http://www.slurrpy.com/wp-content/uploads/2015/09/chicken-burger-800x500.jpg")
 Item.create(name: 'Seared Salmon', description: "This was swimming an hour ago", price: 10.99, category_id: 1, image_url: "http://curves.co.nz/sites/default/files/styles/main_image/public/CUR16617_Curves%20News_posts_Salmon.jpg?itok=KIJwI3oo")
+Item.create(name: 'Lasagna', description: "Just like your granny made it", price: 7.99, category_id: 1, image_url: "http://cdn.thedailybeast.com/content/dailybeast/cheats/2013/02/09/french-gov-vows-to-punish-horsemeat-pushers/jcr:content/image.crop.800.500.jpg/45347925.cached.jpg")
```

Comment:
Line is too long. [282/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.